app/controllers/concerns/hal_actions/actions.rb

```diff
@@ -42,6 +42,8 @@ def destroy_redirect
   def index_options
     valid_params_for_action(:index).tap do |options|
       options[:_keys] = options.keys
+      # TODO: this isn't great
+      options[:represent_with] = Api::PagedCollectionRepresenter
```

Comment:
I _think_ this is okay, as long as we're always going to represent `index` actions with a paged collection.